lib/src/translations.dart

```diff
@@ -13,7 +13,8 @@ class Translations {
 
     /// If we failed to find the key as a nested key, then fall back
     /// to looking it up like normal.
-    returnValue ??= _translations?[key];
+    var maybeValue = _translations?[key];
```

Comment:
Now we do the lookup all the time, even if it's not needed because returnValue != null